scripts/linux/pkg/deb/assets/DEBIAN/postrm

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# postrm script for ravendb
+#
+# see: dh_installdeb(1)
+
+set -e
+
+if [ ! -z "$_DEB_DEBUG" ]; then
+    set -x
+fi
+
+# summary of how this script can be called:
+#        * <postrm> `remove'
+#        * <postrm> `purge'
+#        * <old-postrm> `upgrade' <new-version>
+#        * <new-postrm> `failed-upgrade' <old-version>
+#        * <new-postrm> `abort-install'
+#        * <new-postrm> `abort-install' <old-version>
+#        * <new-postrm> `abort-upgrade' <old-version>
+#        * <disappearer's-postrm> `disappear' <overwriter>
+#          <overwriter-version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    remove|upgrade|failed-upgrade|abort-install|abort-upgrade|disappear)
+    ;;
+
+    purge)
+    
+    rm -v /etc/ravendb/settings.json
```

Comment:
Remove the certificate as well?

Reply:
yes

Reply:
Actually since it's in `/var/lib/ravendb/data` it's removed a line below.